src/main/java/Hnefatafl.java

```diff
@@ -1,3 +1,10 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package hnefatafl;
```

Comment:
Causes class not found for BackgroundPanel, because BackgroundPanel is not in the same package.
Same with tests.